### PR TITLE
Revert CoreDNS Docs

### PR DESCRIPTION
**Pull request by zparnold**
Reverts the CoreDNS docs commits added through merges by: 
* https://github.com/kubernetes/website/pull/10228
* https://github.com/kubernetes/website/pull/10200
* https://github.com/kubernetes/website/pull/10201

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md [hunk 1]
```diff
@@ -28,19 +28,28 @@ DNS is a built-in Kubernetes service launched automatically
 using the addon manager
 [cluster add-on](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/README.md).
 
-As of Kubernetes v1.12, CoreDNS is the recommended DNS Server, replacing kube-dns. However, kube-dns may still be installed by
-default with certain Kubernetes installer tools. Refer to the documentation provided by your installer to know which DNS server is installed by default.
+The running DNS Pod holds 3 containers:
 
+- "`kubedns`": watches the Kubernetes master for changes
```
Comment (tfogo) [inline content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md]: quote marks seem a little unnecessary

File: content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md [hunk 2]
```diff
@@ -146,8 +117,8 @@ kubectl logs --namespace=kube-system $(kubectl get pods --namespace=kube-system
 kubectl logs --namespace=kube-system $(kubectl get pods --namespace=kube-system -l k8s-app=kube-dns -o name | head -1) -c sidecar
 ```
 
-See if there are any suspicious error messages in the logs. In kube-dns, a '`W`', '`E`' or '`F`' at the beginning
-of a line represents a Warning, Error or Failure. Please search for entries that have these
+See if there is any suspicious log. Letter '`W`', '`E`', '`F`' at the beginning
```
Comment (tfogo) [inline content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md]: This is grammatically dubious. The wording in the CoreDNS commits is better.

**Comment by chrisohaver** [comment]
I don't think we should have rolled back the CoreDNS Debug changes (#10201).
These troubleshooting steps are still applicable, even if CoreDNS is not default.
CoreDNS is still widely used (the scalability identified issue wont affect the majority of users).

**Comment by dims** [comment]
Agree with @chrisohaver, @justaugustus @zparnold the 3 commits here look good and relevant for anyone who will be using CoreDNS. irrespective of whether it is the default or now.

Let's please make sure the contents end up in 1.12

cc @tpepper

**Comment by zparnold** [comment]
@dims Ok

**Comment by justaugustus** [comment]
+1 to @chrisohaver and @dims' sentiment. Let's re-revert this.

**Comment by chrisohaver** [comment]
> 3 commits here look good and relevant for anyone who will be using CoreDNS

I think two of the commits  (#10228 and #10200) make specific reference to CoreDNS "being the default" , whereas the troubleshooting/debug changes in #10201 do not.  #10201 does list CoreDNS before kube-dns, but I think that's a fairly minor detail.  I was only recommending we keep the #10201 changes.

**Comment by dims** [comment]
@chrisohaver CoreDNS is the default in kubeadm, so nothing wrong with that statement. in the main k/k repository the only thing that got reverted was a script change for GCE as the script was being used for the scalability test. We have the release note that covers that aspect. So we should be good

**Comment by fturib** [comment]
Was this PR reverted yet ?

**Comment by tfogo** [comment]
@fturib Yes, content from all three PRs are back in the 1.12 branch.

**Comment by fturib** [comment]
Great ! thanks for info.

**Comment by tfogo** [comment]
@fturib can you clarify what you mean, please? Since these three PRs are back in the docs for 1.12, they will already be in 1.13 by default.

**Comment by fturib** [comment]
@tfogo : I am maybe just confused on how the release + master branches are synced ....
- will the branch 1.13 be off of 1.12 ? 
- or 1.12 will be synced into master ?

Right now, I am not seeing the changes in master ... and was thinking 1.13 branch will be created from master.
I just want to be sure that will be in following releases of K8s.

**Comment by tfogo** [comment]
@fturib It will be in the following release.

k/website's branches work a little differently to other repos. `master` is the production branch that the website is served from (currently 1.11). `release-1.12` will be merged into master then `release-1.13` will be cut from master. The reasoning behind this is it's easier for contributors to update the docs and immediately see their improvements published on the website.

**Comment by fturib** [comment]
ok. 
So we are all good then.
Thanks ! :)

**Pull request merged**